runner/src/server/services/payService.ts

```diff
@@ -32,30 +33,24 @@ export class PayService {
     };
   }
 
-  payRequestData(
-    amount: number,
-    reference: string,
-    description: string,
-    returnUrl: string
-  ) {
+  payRequestData(amount: number, description: string, returnUrl: string) {
     return {
       amount,
-      reference,
+      reference: nanoid(10),
```

Comment:
This is different to current - missing FCO prefix. Obviously makes more xgov :). But is that OK for FCDO to drop this? (I'd guess yes, but gotta ask)

Reply:
I'll ask, but this reference is only used for pay. Looking up payment info by reference.. I can't see why having FC(D)O would change anything - it would still have to be the post (or whoever has access to that pay team) that checks it